junction/proposals/forms.py

```diff
@@ -21,10 +21,15 @@
 )
 
 
-def _get_proposal_section_choices(conference):
-    return [(str(cps.id), cps.name)
-            for cps in ProposalSection.objects.filter(
-                conferences=conference, end_date__gt=now())]
+def _get_proposal_section_choices(conference, action="edit"):
+    if action == "create":
+        return [(str(cps.id), cps.name)
+                for cps in ProposalSection.objects.filter(
+                    conferences=conference, end_date__gt=now())]
+    else:
+        return [(str(cps.id), cps.name)
+                for cps in ProposalSection.objects.filter(
+                    conferences=conference)]
```

Comment:
Oh I just read your first comment properly. In that case, I guess this makes sense. Looks good for me.